docs/DeveloperGuide.md

```diff
@@ -525,6 +525,64 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+* 3a. The event has a rate tagging already.
+   * 3a1. System replaces the previous rate with the new rate.
+
+      Use case ends.
+
+**Use case: UC9 - Add Contact**
+
+**MSS**
+
+1.  User requests to add a contact to the list of contact.
+2.  System adds the contact and displays information of the contact added.
+
+   Use case ends.
+
```

Comment:
Are we still doing a `addcontact` separately at v1.1?

Reply:
Should `addcontact` be to link a contact, either new or already in the system, to the event, instead of adding the a new contact to the system?